__tests__/app.test.js

```diff
@@ -106,21 +106,22 @@ describe("/api/articles", () => {
         .expect(200)
         .then(({ body: { articles } }) => {
             expect(articles).toBeInstanceOf(Array);
-            if (articles.length === 0){
-                return;
-            }
-            articles.forEach((article) => {
-                expect(article).toMatchObject({
-                    author: expect.any(String),
-                    title: expect.any(String),
-                    article_id: expect.any(Number),
-                    topic: expect.any(String),
-                    created_at: expect.any(String),
-                    votes: expect.any(Number),
-                    article_img_url: expect.any(String),
-                    comment_count: expect.any(Number),
+            expect(articles).toHaveLength(13);
+
+            if (articles.length > 0) {
```

Comment:
You don't have to include this here ☺️  It's the same logic as you have on line 109, it'll always run if 109 passes because the length will > 0 (13!)